app/views/info/_per_page_metrics.html.erb

```diff
@@ -0,0 +1,42 @@
+<% if !per_page_metrics.empty? %>
+  <h2 class="govuk-heading-l">Metrics for individual pages</h2>
+  <div id="per-page-metrics">
+    <%
+      rows = []
+      per_page_metrics.each do |t, a|
```

Comment:
Could `t` and `a` be more descriptively named please?

Reply:
That's sensible. It's not obvious (because I renamed the file) but that wasn't my change. I've made some educated guesses as to what `t` and `a` are, though.